Corona-Warn-App/src/main/res/layout/home_statistics_cards_local_incidence_layout.xml

```diff
@@ -0,0 +1,121 @@
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:contentDescription="@null"
+            android:importantForAccessibility="no"
+            android:paddingStart="0dp"
+            android:paddingEnd="@dimen/spacing_small"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@id/flow_layout"
+            app:srcCompat="@drawable/ic_statistics_local_incidence" />
+
+        <TextView
+            android:id="@+id/title"
+            style="@style/headline5"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_normal"
+            android:layout_marginTop="@dimen/spacing_normal"
+            android:text="@string/statistics_card_local_incidence_title"
+            app:layout_constraintEnd_toStartOf="@id/overflow_menu_button"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/location_label"
+            style="@style/StatisticsCardTitleCaption"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_normal"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/title"
+            tools:text="Potsdam" />
+
+        <TextView
+            android:id="@+id/primary_label"
+            style="@style/StatisticsCardValueLabel"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            tools:text="Bis gestern" />
+
+        <TextView
+            android:id="@+id/primary_value"
+            style="@style/StatisticsCardPrimaryValue"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:focusable="true"
+            android:includeFontPadding="false"
+            tools:text="98,9" />
+
+        <TextView
+            android:id="@+id/secondary_label"
+            style="@style/StatisticsCardValueLabel"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:paddingTop="@dimen/spacing_normal"
+            android:text="@string/statistics_card_local_incidence_value_description" />
+
+        <de.rki.coronawarnapp.statistics.ui.TrendArrowView
+            android:id="@+id/trend_arrow"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_tiny"
+            app:layout_constraintBottom_toBottomOf="@id/primary_value"
+            app:layout_constraintStart_toEndOf="@id/primary_value"
+            app:layout_constraintTop_toTopOf="@id/primary_value" />
+
+        <ImageButton
+            android:id="@+id/overflow_menu_button"
```

Comment:
Do we need an accessibility label for the menu button?

Reply:
the contact diary card i took it from did not have any but we could probably add one